### PR TITLE
Update Authors

### LINKED ISSUE #5: Add authors
Add missing authors to:
- [x] .zenodo.json
- [x] CITATION.cff

### PR DESCRIPTION
**Pull request by giovannivolpe**

### CODE REVIEW

**Comment by giovannivolpe** [comment]
@hanzh346 @bzufiriaki @JiaweiSun0512 please check the changes with your author information.

@hanzh346 @JiaweiSun0512 please add your ORCID (if you don't have one, create it).

File: CITATION.cff [hunk 1]
```diff
@@ -39,8 +39,28 @@ authors:
...
     affiliation: Karolinska Institute
     orcid: 'https://orcid.org/0000-0003-2077-5252'
+  - given-names: Hang
+    family-names: Zhao
+    email: hang.zhao@physics.gu.se
+    affiliation: Gothenburg University
+    orcid: 'https://orcid.org/'
+  - given-names: Blanca
+    family-names: Zufiria Gerbolés
+    email: blanca.zufiria@ki.se
```
Comment (bzufiriaki) [inline CITATION.cff]: email: blanca.zufiria.gerboles@ki.se

**Review by bzufiriaki — CHANGES_REQUESTED** [review]

File: CITATION.cff (hunk 1 above)
Comment (giovannivolpe) [reply CITATION.cff]: changed email

File: .zenodo.json [hunk 2]
```diff
@@ -39,6 +39,21 @@
...
+            "affiliation": "Gothenburg University",
+            "name": "Zhao, Hang"
+        },
+        {
+            "orcid": "0000-0001-7756-630X",
+            "affiliation": "Karolinksa Institute",
+            "name": "Zufiria Gerbolés, Blanca"
+        },
+        {
+            "orcid": "",
```
Comment (JiaweiSun0512) [inline .zenodo.json]: 0000-0002-2986-2734

File: CITATION.cff [hunk 3]
```diff
@@ -39,8 +39,28 @@ authors:
...
+  - given-names: Blanca
+    family-names: Zufiria Gerbolés
+    email: blanca.zufiria.gerboles@ki.se
+    affiliation: Karolinska Institute
+    orcid: 'https://orcid.org/0000-0001-7756-630X'
+  - given-names: Jiawei
+    family-names: Sun
+    email: jiawei.sun@ki.se
+    affiliation: Karolinska Institute
+    orcid: 'https://orcid.org/'
```
Comment (JiaweiSun0512) [inline CITATION.cff]: https://orcid.org/0000-0002-2986-2734

**Pull request merged**